### DISCUSSION TITLE
AsyncAPI

### DISCUSSION THREAD

**Discussion by thelinuxlich** [scalar/scalar#1124] [Ideas]
Swagger supports both OpenAPI and AsyncAPI, is it on Scalar radar?

**Comment by marclave** [comment]
@thelinuxlich it is on our radar! 

We should probably add items like this to a GitHub issue and have a tag as planned ✨ i do need to see if GitHub has a better of way of showing planned inside of discussion but maybe that's when it moves to issues 😅 

Most likely Q2!

**Comment by romgerman** [comment]
Support of AsyncAPI would be great because AsyncAPI doesn't really have a good looking documentation generator.

**Reply by marclave** [reply]
@romgerman appreciate the comment! We can make sure this happens in Q2 :) will update this discussion once we kick that off ✨

**Reply by jdkajewski** [reply]
@marclave, any update on whether this made it into planning? Would love to be able to display async api docs using scalar.

**Reply by isamauny** [reply]
Any news on this ?

**Comment by hanspagel** [comment]
AsyncAPI support is there. :)
